packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -730,8 +780,16 @@ function* orderPlaylistAsync(action) {
       })
     }
   }
+  playlist.playlist_contents.track_ids =
+    updatedPlaylist.playlist_contents.track_ids
 
```

Comment:
Could you just remove this and pass updatedPlaylist below? (Personally prefer not mutating objects directly, tends to be bug prone)